pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -2102,8 +2108,8 @@ def allocate_budget_to_maximize_response(
             The total budget to be allocated.
         time_granularity : str
             The granularity of the time periods (e.g., 'daily', 'weekly', 'monthly').
-        num_days : int
-            The number of days over which the budget is to be allocated.
+        horizon : int
+            The number of time units over which the budget is to be allocated.
```

Comment:
Maybe refer to these as "periods" because that's what the description of `time_granularity` refers to (assuming that `horizon` is measured in units of `time_granularity`)?  So:

```
horizon : int
    The number of time periods over which the budget is to be allocated.
```

Reply:
Correct, I'll replace ✌🏻

Reply:
Also, I am not sure what standard procedure for the project is given that we're at v0.8.0 at this point, but because this is a breaking change, I would typically recommend throwing a warning if the old argument name is used (I mean, usually you would want through a whole deprecation cycle with  future warnings, etc. but probably not necessary for versions < 1.0).